test/fixtures/handlebars/www/page-handlebars.handlebars

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html lang="en">
+  <body>
+    <h1>Hello There</h1>
+    <p>This template had the extension .handlebars</p>
+  </body>
+</html>
```

Comment:
We might want to include some minor handlebars expression in all 3 of these .ext templates, just to ensure the handlebars processor is actually invoked and that we're not just testing P9K's ability to rename files. :smile_cat: